src/client/styles/components/_sidebar.scss

```diff
@@ -1,18 +1,82 @@
 .sidebar {
+  h2#list-title {
+    color: $nypl-red;
+    display: block;
+    font-size: 1.2rem;
+    font-weight: 700;
+   @include min-screen($mobile-breakpoint) {
+      display: none;
+    }
+  }
+  h3 {
+    font-size: 1rem;
+    font-weight: 200;
+    margin: 16px 0;
+
+    svg {
+      float: left;
+      margin: -4px 0 0 -5px;
+    }
+  }
+  nav {
+    @include min-screen($mobile-breakpoint) {
+      border-top: 0;
+    }
+  }
+  fieldset {
+    border: 0;
+    padding: 0.35rem 0;
+    margin: 0 0 10px 0;
+    label {
+      font-size: 0.75rem;
+      vertical-align: sub;
+    }
+    select {
+      /* Hides -webkit-appearance so we can use costmized dropdown icon*/
+      -webkit-appearance: none;
+      appearance: none;
+      /* The costmized dropdown icon*/
+      background: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI3LjY5OTA1IiBoZWlnaHQ9IjMuOTg3MTMiIHZpZXdCb3g9IjAgMCA3LjY5OTA1IDMuOTg3MTMiPgogIDx0aXRsZT5kb3duLjIud2VkZ2U8L3RpdGxlPgogIDxwb2x5bGluZSBwb2ludHM9IjcuMzk1IDAuMzYgMy44NSAzLjM3IDAuMzA0IDAuMzU4IiBmaWxsPSIjZmZmIiBzdHJva2U9IiMyMzFmMjAiIHN0cm9rZS1taXRlcmxpbWl0PSIxMCIgc3Ryb2tlLXdpZHRoPSIwLjk0MDUiLz4KPC9zdmc+) no-repeat 95% #fff;
```

Comment:
Where did this come from?

Reply:
https://nypl.github.io/design-toolkit/sections/forms.html#omniSearch

I copied the same style of the dropdwon from omni search form

Reply:
I think I see what's happening. You should use the class names and not copy/paste from the toolkit. That should work.

Reply:
I have some concern about that. Let's talk about this a bit.

Reply:
yeah I have looked into it. I have some concerns about it.

1. The class names for it are `nypl-omnisearch` and `nypl-omni-fields`. They don't really describe the form correctly for its role.
2. The styles not only include the drop down icon, but also other features of the omni search. So if we use it, we still need to add some other styles to adjust them.

I feel the better way is to customize the styles now and maybe @ricardoom can come up a new style sample for this new dropdown on the toolkit? Then I can switch it.